crates/wit-component/tests/components/adapt-inject-stack-with-reallocing-adapter/component.wat

```diff
@@ -0,0 +1,158 @@
+(component
+  (type (;0;)
+    (instance
+      (type (;0;) (func (result "a" u32) (result "b" u32)))
+      (export (;0;) "get-two" (func (type 0)))
+    )
+  )
+  (import "new" (instance (;0;) (type 0)))
+  (core module (;0;)
+    (type (;0;) (func (result i32)))
+    (type (;1;) (func (param i32 i32 i32 i32) (result i32)))
+    (import "old" "get_sum" (func (;0;) (type 0)))
+    (func $cabi_realloc (;1;) (type 1) (param i32 i32 i32 i32) (result i32)
+      (local i32)
+      i32.const 0
+      local.get 0
+      i32.ne
+      if ;; label = @1
+        unreachable
+      end
+      i32.const 0
+      local.get 1
+      i32.ne
+      if ;; label = @1
+        unreachable
+      end
+      i32.const 65536
+      local.get 3
+      i32.ne
+      if ;; label = @1
+        unreachable
+      end
+      i32.const 1
+      memory.grow
+      local.tee 4
+      i32.const -1
+      i32.eq
+      if ;; label = @1
+        unreachable
+      end
+      local.get 4
+      i32.const 16
+      i32.shl
+    )
+    (memory (;0;) 1)
+    (export "memory" (memory 0))
+    (export "cabi_realloc" (func $cabi_realloc))
+  )
+  (core module (;1;)
+    (type (;0;) (func (param i32)))
+    (type (;1;) (func (result i32)))
+    (type (;2;) (func (param i32 i32 i32 i32) (result i32)))
+    (type (;3;) (func))
+    (import "env" "memory" (memory (;0;) 0))
+    (import "new" "get-two" (func $get_two (;0;) (type 0)))
+    (import "__main_module__" "cabi_realloc" (func (;1;) (type 2)))
```

Comment:
As a very minor nit, could this import, which is synthesized by the gc pass, get a name `cabi_realloc`?